system-tests/005/__snapshots__/005.test.js.snap

```diff
@@ -11765,169 +11765,169 @@ Array [
         "text": "Expected line length to be no more than 20 characters (max-line-length)",
       },
       Object {
-        "column": 5,
-        "line": 146,
+        "column": 25,
+        "line": 145,
```

Comment:
@m-allanson I actually didn't even get to the stage of checking if the column number was correct, I've only really tested line numbers, and it's been clear that the line numbers referenced are not 20 characters in length (based on this PRs settings)